src/plugins/installer.py

```diff
@@ -0,0 +1,99 @@
+import os
+from pathlib import Path
+
+from helperFunctions.install import read_package_list_from_file, run_cmd_with_logging
+
+
+class AbstractPluginInstaller:
+    # Even if some functoins don't need self we want to have them nicely
+    # grouped in this class
+    # pylint:disable=no-self-use
```

Comment:
It should also be fine to convert the methods to static methods in this class, since overriding them with non-static methods should work without problems.